spec/cr-image/format/png_spec.cr

```diff
@@ -10,7 +10,7 @@ Spectator.describe CrImage::Format::PNG do
         io = IO::Memory.new
         image.to_png(io)
 
-        expect_digest(io.to_s).to eq "c9450089873a4bbe98a2aa54f1bea959915ba088"
+        expect_digest(io.to_s).to eq "8635306d7a0201533e8812ebf3671b9c2e31c4b0"
```

Comment:
Visually the images are identical, so I'm assuming a slight bit twiddle somewhere as the difference